Doc/whatsnew/3.8.rst

```diff
@@ -529,6 +529,13 @@ Other Language Changes
 
   (Contributed by Jörn Heissler in :issue:`35224`.)
 
+* The :meth:`object.__reduce__` method can now return a tuple from two to
+  six elements long. Formerly, five was the limit.  The new, optional sixth
+  element is a callable with a ``(obj, state)`` signature.  This allows the
+  direct control over the state-updating behavior of a specific object.  If
+  not *None*, this callable will have priority over the object's
+  :meth:`~__setstate__` method.
```

Comment:
`~` is not needed here. It could be ``` :meth:`~object.__setstate__` ```, but maybe ``` :meth:`__setstate__` ``` is enough.

Reply:
```
:meth:`__setstate__`
```
IIRC, the above does not correctly generate an inline link. This applies to the other dunder methods and attributes inherited from object as well, it has to be:
```
:meth:`object.__setstate__`
```
Usually, I've used:
```
:meth:`~object.__setstate__`
```
since only the `__setstate__` part is relevant for the context of the readers.